nutribest.com/Admin-Side/Orders/Shipped.jsx

```diff
@@ -1,277 +0,0 @@
-            if (UData[i].Orders[j].Order_status == "Shipped") {
-              UData[i].Orders[j].Order_status = "Completed";
-            }
-          }
-        }
-      }
-    }
-    for (let i = 0; i < UData.length; i++) {
-      if (UData[i].id == uID) {
-        try {
-          let res = await axios.patch(
-            `http://localhost:8080/User-Details/${uID}`,
-            UData[i]
-          );
-          GetUserOrderDetails();
-          toast({
-            title: "Updated Successfully.",
-            description: "Product Marked as Completed Successfully",
-            status: "success",
-            duration: 3000,
-            isClosable: true,
-          });
-        } catch (error) {
-          toast({
-            title: "Invalid Request",
-            description: "Please Try Again",
-            status: "error",
-            duration: 3000,
-            isClosable: true,
-          });
-        }
-      }
-    }
-    // console.log(UData);
-  };
-
-  // const GetUserOrderDetails = async () => {
-  //   let res = await axios.get(`http://localhost:8080/User-Details`);
-  //   setUserDetails(res.data);
-  //   return res.data;
-  // };
-
-  // useEffect(() => {
-  //   GetUserOrderDetails();
-  // }, []);
-  return (
-    <>
-      <Box
-        display={{ base: "none", md: "flex" }}
-        justifyContent={"space-between"}
-        alignItems={"center"}
-        boxShadow="rgba(0, 0, 0, 0.4) 0px 1px 4px, rgba(0, 0, 0, 0.3) 0px 5px 10px -1px, rgba(0, 0, 0, 0.2) 0px -1px 0px inset"
-        p="10px"
-        mb={"20px"}
-      >
-        <Box
-          width={{ base: "10%", md: "12%" }}
-          fontSize={{ base: "12px", md: "11px", lg: "14px" }}
-        >
-          <Text>NAME</Text>
-        </Box>
-        <Box
-          width={{ base: "10%", md: "17%" }}
-          fontSize={{ base: "12px", md: "11px", lg: "14px" }}
-        >
-          <Text>ADDRESS</Text>
-        </Box>
-        <Box
-          width={{ base: "10%", md: "10%", lg: "7%" }}
-          fontSize={{ base: "12px", md: "11px", lg: "14px" }}
-        >
-          <Text>IMAGE</Text>
-        </Box>
-
-        <Box
-          width={{ base: "15%", md: "28%" }}
-          fontSize={{ base: "12px", md: "11px", lg: "14px" }}
-        >
-          <Text>PRODUCTS</Text>
-        </Box>
-
-        <Box
-          width={{ base: "10%", md: "11%" }}
-          fontSize={{ base: "12px", md: "11px", lg: "14px" }}
-          textAlign={"center"}
-        >
-          <Text>AMOUNT</Text>
-        </Box>
-        <Box
-          width={{ base: "10%", md: "10%" }}
-          fontSize={{ base: "12px", md: "11px", lg: "14px" }}
-          textAlign={"center"}
-        >
-          <Text>DATE</Text>
-        </Box>
-        <Box
-          width={{ base: "20%", md: "15%" }}
-          fontSize={{ base: "12px", md: "11px", lg: "14px" }}
-          textAlign={"center"}
-        >
-          <Text>STATUS</Text>
-        </Box>
-      </Box>
-      {userDetails.map((user) =>
```

Comment:
I have seen this type of code in multiple files. Ex: Orders/Completed, Order/Pending etc. Can we please move this type of similar code to a new component and reuse it everywhere by passing different values via props?